patroni/postgresql/mpp/__init__.py

```diff
@@ -93,75 +93,154 @@ def __init__(self, postgresql: 'Postgresql', config: Dict[str, Union[str, int]])
 
     @abc.abstractmethod
     def handle_event(self, cluster: Cluster, event: Dict[str, Any]) -> None:
-        pass
+        """Handle an event sent from a worker node.
+
+        :param cluster: the currently known cluster state from DCS.
+
+        :param config: the event to be handled.
+        """
 
     @abc.abstractmethod
-    def sync_pg_dist_node(self, cluster: Cluster) -> None:
-        pass
+    def sync_meta_data(self, cluster: Cluster) -> None:
```

Comment:
It seems we are doing more than just docstrings and unit tests :)

Reply:
Yeah, do you think we should split this in a future PR? Also want to here from @CyberDem0n

Reply:
got it.